pylops/waveeqprocessing/oneway.py

```diff
@@ -14,51 +15,11 @@
 from pylops.utils import dottest as Dottest
 from pylops.utils.backend import to_cupy_conditional
 from pylops.utils.tapers import taper2d, taper3d
+from pylops.utils.typing import DTypeLike, NDArray
 
 logging.basicConfig(format="%(levelname)s: %(message)s", level=logging.WARNING)
 
 
-def _phase_shift(phiin, freq, kx, vel, dz, ky=0, adj=False):
-    """Phase shift extrapolation for single depth level in 2d/3d constant
-    velocity medium
```

Comment:
I didnt't fully understand where this went

Reply:
I just realized that this was never used, in fact the very same code was repeated in the `_PhaseShift` init method. I don't recall how this happened, likely because at some point I was trying to write a one-way migration operator using this function but I eventually never pushed it and this function stayed there... ;)